nessie/api/status_controller.py

```diff
@@ -34,17 +34,32 @@
 from nessie.lib.http import tolerant_jsonify
 
 PUBLIC_CONFIGS = [
+    'ASC_ATHLETES_API_URL',
+    'CAL1CARD_PHOTO_API_URL',
+    'CALENDLY_BASE_API_URL',
+    'CALENDLY_FETCH_DAYS_AHEAD',
+    'CALENDLY_FETCH_DAYS_BEHIND',
+    'CANVAS_DATA_HOST',
+    'CANVAS_HTTP_URL',
+    'CAS_SERVER',
     'CURRENT_TERM',
     'EARLIEST_ACADEMIC_HISTORY_TERM_ID',
     'EARLIEST_LEGACY_TERM',
     'EARLIEST_TERM',
     'EB_ENVIRONMENT',
     'EMAIL_SYSTEM_ERRORS_TO',
     'FUTURE_TERM',
+    'HISTORY_DEPT_NOTES_DEFAULT_ADVISOR_UID',
     'JOB_SCHEDULING_ENABLED',
+    'LOGGING_LEVEL',
+    'LOGGING_PROPAGATION_LEVEL',
     'NESSIE_ENV',
+    'PIAZZA_API_URL',
     'WORKER_QUEUE_ENABLED',
     'WORKER_THREADS',
+    'YCBM_BASE_URL',
+    'YCBM_FETCH_DAYS_AHEAD',
+    'YCBM_FETCH_DAYS_BEHIND',
 ]
```

Comment:
...And, we expose a few more configs in the UI. Useful when smoke-testing a deploy.